therapy_app/client.py

```diff
@@ -0,0 +1,175 @@
+from django.db.models import Q
+from rest_framework.views import APIView, Response, status
+from . import models
+from .authentication_classes import CsrfExemptSessionAuthentication
+from .permissions import *
+from . import serializers
+
+
+# Create your views here.
+class SearchTherapistView(APIView):
+    """
+    This APIView takes GET request only to show available therapists to clients.
+    This only shows those therapists that are not actively mapped to the requesting client.
+    """
+    authentication_classes = (CsrfExemptSessionAuthentication,)
+    permission_classes = [IsClient]
+
+    def get(self, request, format=None):
+        """
+        This method will return the list of therapists that are not actively mapped to the requesting client.
+        """
+        therapists = models.User.objects.filter(is_deleted=False, type='Therapist').exclude(
+            Q(mapping_therapist__client=request.user) & Q(mapping_therapist__is_deleted=False))
```

Comment:
Like the fact that you're excluding current therapists and removed therapists from this list - this shows your ownership for the task!